app/src/main/res/xml-eu/ext_kbd_bottom_row_ng.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Ok, thanks for your help. In your opinion, which one is the best option? Should I create an additional bottom row with hyphen or should I wait until you explore the possibility to add all those bottom rows as a part of the Basque language pack?